src/main/java/org/myrobotlab/service/MouthControl.java

```diff
@@ -18,79 +20,31 @@
 public class MouthControl extends Service {
 
   // TODO: remove Peer & Make it attachable between generic servoControl & SpeechSynthesis
-  
+
   private static final long serialVersionUID = 1L;
   public final static Logger log = LoggerFactory.getLogger(MouthControl.class.getCanonicalName());
   public int mouthClosedPos = 20;
   public int mouthOpenedPos = 4;
   public int delaytime = 75;
   public int delaytimestop = 150;
   public int delaytimeletter = 45;
-  transient Servo jaw;
-  transient Arduino arduino;
-  transient SpeechSynthesis mouth;
-  
-  @Deprecated
-  public boolean autoAttach = true;
+  transient private ServoControl jaw;
+  transient private SpeechSynthesis mouth;
 
   public MouthControl(String n) {
     super(n);
-    jaw = (Servo) createPeer("jaw");
-    arduino = (Arduino) createPeer("arduino");
-    mouth = (SpeechSynthesis) createPeer("mouth");
-
-    // TODO: mouth should probably implement speech synthesis.
-    // in a way of speaking, one day, people may be able to read the lips
-    // of the inmoov.. so you're synthesising speech in a mechanical way.
-    // similar to sign language maybe?
-    subscribe(mouth.getName(), "publishStartSpeaking");
-    subscribe(mouth.getName(), "publishEndSpeaking");
   }
 
-  // FIXME make interface
-  public boolean connect(String port) throws Exception {
```

Comment:
changing/removing this method name will probably break a bunch of scripts. but I think it's better to start moving towards the attach methods.